DotnetApi/Controllers/ApiController.cs

```diff
@@ -15,96 +15,457 @@ public class ApiController : ControllerBase
     {
         private readonly ILogger<ApiController> _logger;
         private readonly PostgresContext _dbContext;
+        private readonly AppService _appService;
+        private readonly AttributeService _attributeService;
+        private readonly AttributeTypeService _attributeTypeService;
         private readonly EntityService _entityService;
+        private readonly EntityTypeService _entityTypeService;
+        private readonly RecordService _recordService;
 
         public ApiController(

```

Comment:
Thoughts on breaking up this controller into multiple classes? Like one for records and another for entityType?

Reply:
@Tony-T-R-A-N addressed